airbyte-integrations/connectors/source-greenhouse/source_greenhouse/manifest.yaml

```diff
@@ -83,38 +78,760 @@ definitions:
       class_name: source_greenhouse.components.GreenHouseSlicer
       request_cursor_field: "created_after"
       cursor_field: "applied_at"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
```

Comment:
noting the schema definitions will be extracted in their own block the first time the connector is edited with the builder. Not something to block on, but it would be nice to have a script that formats the manifest exactly how the builder likes it. 

CC @lmossman

Reply:
Happy to work on that for future migrations. Have an example?

Reply:
Here's an example: https://gist.github.com/lmossman/1a01b738f2363c896bf557b006839240